src/core-layers/hexagon-cell-layer/hexagon-cell-layer-vertex-64.glsl.js

```diff
@@ -49,10 +49,10 @@ void main(void) {
   mat2 rotationMatrix = mat2(cos(angle), -sin(angle), sin(angle), cos(angle));
 
   vec2 rPos = rotationMatrix * positions.xz;
-  vec2 rNorm = rotationMatrix * normals.xz;
+  vec2 rNorm = rotationMatrix * normals.xz; // the hexagon cells has y axis as the vertical axis
```

Comment:
My sense is that we should change the geometry to be consistent with a geospatial coordinate system which we clearly document to be Z = up, and adjust the shader accordingly. Don't mind if land this as a quick fix, but in that case it would be good to open an issue to make sure we eventually do it right.

Reply:
+1. At lease this should be rolled into the rotation matrix.